.github/ISSUE_TEMPLATE/error-report.md

```diff
@@ -0,0 +1,42 @@
+---
+name: Error report
+about: Used to report production errors seen in AMP Error Reporting.
+title: "\U0001F6A8 Error: [error message]"
+labels: 'Type: Error Report'
+assignees: ''
+
+---
+
+<!--
+Please only file error reports from AMP Error Reporting here.
+Replace/remove all of the text in brackets, including this text.
+-->
+
+Details
+---
+
+**Error report:** [link](go/ampe/<Error_ID>)
+**First seen:** <First seen date> <!--
+Select "1 day" on the error details page for the error and record the
+occurrences.
+-->
+**Frequency:** ~ <Daily occurrences>/day
```

Comment:
These are showing up on the same line in HTML rendered view. Maybe they need blank lines in between?

Reply:
They render properly when GitHub creates an issue from it, so I think it's just an artifact of the GitHub preview in review?

Reply:
(I spent some time fiddling with that spacing)